apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -103,15 +103,19 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
         uint256 yea;                                        // Voting power for
```

Comment:
Note that this change (75ede7c916aab8d5c6bd1c25fb228ec1574fbb74) (and the associated commit fixing the gas test: 03e1f71) appear to actually be increasing the gas.

This is as micro-optimizing as it gets, but I wanted to prefer saving gas on votes.

-----

The full analysis:

- Switching the Vote's struct order like this increases the cost of challenging by ~19k gas
- Based on the change below with pre-fetching `pauseDuration`:
  - When voting before a challenge, this adds ~50-100 gas per vote (and the same amount if representative is voting)
  - On challenged votes, this saves ~750 gas per vote (and another ~750 saved if voting representative is voting)

On a rough calculation, this means you would need ~25 votes to break even with the challenger, but the challenger is also financially motivated and should be able to afford more gas.